scripts/compile.mjs

```diff
@@ -9,30 +9,39 @@ import * as url from 'node:url';
 const __dirname = path.dirname(url.fileURLToPath(import.meta.url));
 
 // =========================================
-//         rename vite outputs
+//     rename styles generated by Vite
 // =========================================
 
-await Promise.all([
-  fs.promises.rename(
-    path.resolve(__dirname, '../dist/higlass.umd.js'),
-    path.resolve(__dirname, '../dist/hglib.js'),
-  ),
-  fs.promises.rename(
-    path.resolve(__dirname, '../dist/style.css'),
-    path.resolve(__dirname, '../dist/hglib.css'),
-  ),
-]);
+await fs.promises.rename(
+  path.resolve(__dirname, '../dist/style.css'),
+  path.resolve(__dirname, '../dist/hglib.css'),
+);
 
 // =========================================
-//        minify (dist/hglib.min.js)
+//  prod (un)minified UMD (hglib/hglib.min)
 // =========================================
 
+// we transform the contents of the vite build
+const viteUmdOutput = path.resolve(__dirname, '../dist/higlass.umd.js');
```

Comment:
Will this mess up people that use unpkg to import higlass and rely on `hglib.js`?

Reply:
I think `hglib.js` and `hglib.min.js` will still be available.

Reply:
@alexander-veit is correct. The umd output from Vite needs extra compilation to to make `hglib.js` and `hglib.min.js` so that we are backwards compatible. These will still expose the same API for use from `unpkg`. The ESM output from Vite is what bundlers get.